lib/wisper/registration/object.rb

```diff
@@ -8,7 +8,7 @@ def initialize(listener, options)
       super(listener, options)
       @with   = options[:with]
       @prefix = ValueObjects::Prefix.new options[:prefix]
-      @allowed_classes = Array(options[:scope]).map(&:to_s).to_set
+      @allowed_classes = Array(options[:scope]).to_set
```

Comment:
You can use an array here instead. It will be faster to `&` with `constantized_classes`

    @allowed_classes = Array(options[:scope])